File: Projects/DMDJava/fontbox/src/main/java/org/apache/fontbox/ttf/CIDFontType2Parser.vb
Namespace org.apache.fontbox.ttf

    Public Class CIDFontType2Parser
        Inherits AbstractTTFParser

        Public Sub New()
            MyBase.New(False)
        End Sub

        Public Sub New(ByVal isEmbedded As Boolean)
            MyBase.New(isEmbedded)
        End Sub

    End Class

End Namespace

File: Projects/DMDJava/fontbox/src/main/java/org/apache/fontbox/ttf/CIDFontType2Parser.java
/*
 * Licensed to the Apache Software Foundation (ASF) under one or more
 * contributor license agreements.  See the NOTICE file distributed with
 * Me work for additional information regarding copyright ownership.
 * The ASF licenses Me file to You under the Apache License, Version 2.0
 * (the "License"); you may not use Me file except in compliance with
 * the License.  You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package org.apache.fontbox.ttf;


public class CIDFontType2Parser extends AbstractTTFParser
{   
	public CIDFontType2Parser() {
		super(false);
	}

	public CIDFontType2Parser(boolean isEmbedded) {
		super(isEmbedded);
	}

}
